frontend/src/app/main/ui/workspace/sidebar/options/menus/component.scss

```diff
@@ -10,25 +10,47 @@
     @include flexColumn;
     .component-wrapper {
       display: flex;
-      gap: $s-4;
+      margin: 0 $s-4 0 $s-8;
       .component-name-wrapper {
         @extend .asset-element;
         @include flexRow;
         flex-grow: 1;
+        height: 100%;
+        width: 100%;
+        flex-wrap: wrap;
+        padding: 0 0 0 $s-12;
+        margin-top: $s-8;
+
+        &.with-main {
+          padding-bottom: $s-12;
+        }
+
         .component-icon {
           @include flexCenter;
           height: $s-24;
           width: $s-24;
           svg {
             @extend .button-icon;
-            stroke: var(--icon-foreground);
+            stroke: var(--color-gray-20);
```

Comment:
Don't use color defs directly, create a desing token fot it, then use that design token